docs/src/Button.doc.js

```diff
@@ -9,281 +9,287 @@ import PageHeader from './components/PageHeader.js';
-        href: 'combinations',
+        description:
+          'Display a button in different sizes. Size only changes top & bottom padding modifying the component height: sm (32px), md (40px), lg (48px).',
+        href: 'height',
       },
       {
         name: 'text',
         type: 'string',
         required: true,
+        defaultValue: null,
+        description: buildDescription([
+          'Text to render inside the button to convey the function and purpose of the button. The button text has a fix size.',
+          'Accessibility: Screen readers read the `accessibilityLabel` prop, if present, instead of the `text` prop. See `accessibilityLabel` for more info.',
+        ]),
+        href: 'basic-button',
       },
       {
         name: 'textColor',
-        type: `"blue" | "red" | "darkGray" | "white"`,
-        description:
-          'The color for the text shown inside the Button. Note: should only be used for "white" buttons',
+        type: `'white' | 'darkGray' | 'blue' | 'red'`,
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Primary colors to apply to the text inside the Button.',
+          'Text colors are automatically paired with the defined button `color`.',
+          '`textColor` should only be used when using a white background color; it may become necessary to manually override the text color to match that of the parent`s background. Do not override the text color for any button besides those with white background as it breaks design standards.',
+        ]),
         href: 'textColor',
       },
       {
         name: 'type',
-        type: `"submit" | "button"`,
+        type: `'submit' | 'button' | 'link'`,
+        required: false,
         defaultValue: 'button',
-        href: 'types',
+        description: buildDescription([
+          'Select a button variant:',
+          '-`button`: Use to render simple push buttons with no default behavior and control custom functionality inside the `onClick` callback. The button is rendered as an button or  `<button> tag`.',
+          '-`submit`: Use to submit forms.  The button is inside/associated with a form. The button is rendered as an button or `<button> tag`.',
+          '-`link`: Use for buttons to act like links. The button is rendered as an anchor or `<a> tag`.',
+          'Required with submit-type and link-type buttons.',
+        ]),
+        href: 'type',
+      },
+      {
+        name: 'selected',
+        type: 'boolean',
+        required: false,
+        defaultValue: false,
+        description: buildDescription([
+          'Control the "selected" state of a button component to toggle binary states.',
+          'Optional with button-type buttons.',
+        ]),
+        href: '',
+      },
+      {
+        name: 'href',
+        type: 'string',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Specify a redirect url.',
```

Comment:
@AlbertCarreras not really a redirect, let's rephrase this as `Specify a link URL.`